src/components/CheckboxField/CheckboxField.tsx

```diff
@@ -27,10 +29,6 @@ export interface Props {
    * Used as helper text or error message
    */
   fieldNote?: string | ReactNode;
-  /**
-   * Toggles the visibility of the form control legend
-   */
-  hideLegend?: boolean;
```

Comment:
This has been removed since the dev can just not include the subcomponent `<FieldsetLegend>`

Reply:
What was it even doing?

Reply:
It was applying that `u-is-vishidden` class that hides the legend on the screen but is available for screen readers. I think simply applying `aria-label` or any similar aria naming attributes on the `fieldset` is a better alternative since it is more native